source/manual/set-up-aws-account.html.md

```diff
@@ -66,6 +66,8 @@ Add yourself to a lists of users found in [the data for the infra-security proje
 - `govuk-platformhealth-powerusers`: as above but for members of the GOV.UK Platform Health team
 - `govuk-users`: anyone else who needs integration access on GOV.UK
 
+> Note: There is a limit on the number of people that can be in each group. If you find that the limit has been hit, try and identify any users who no longer need access and can be removed. Otherwise, a new group will need to be created.
```

Comment:
I have ideas for how to change our AWS roles to be better, but these docs are good for now. Though it's potentially irrelevant here because people setting up their account won't know whether the list is too long - it only surfaces on deploy. Something to look into is how we can surface this... but I'm going off-piste...

Reply:
I'm not sure why I made a note of this, but I think it came up a year ago or so when a tech lead was trying to add someone in their team... so perhaps it would at least be useful to know why it might not be working? But I'm not sure how it got surfaced to them.